package.json

```diff
@@ -11,6 +11,7 @@
     "_rushInstall": "node common/scripts/install-run-rush.js install --purge",
     "_rushBuild": "node common/scripts/install-run-rush.js build",
     "_rushBuildToFabric": "node common/scripts/install-run-rush.js build --to @uifabric/fabric-website-resources",
+    "_rushBuildToExperiments": "node common/scripts/install-run-rush.js build --to experiments",
```

Comment:
You should probably include `--verbose` and possibly `--lint` and/or `--production` to match the other commands. The output is not very useful without verbose mode.

Reply:
@ecraig12345 I was actually following the syntax of `_rushBuildToFabric`, which does not have either `--lint` nor `--verbose`, but we might want to add them to that command too